src/components/Executions/ExecutionFilters.tsx

```diff
@@ -57,7 +57,19 @@ export const ExecutionFilters: React.FC<{
   clearCharts?: () => void;
   showArchived?: boolean;
   onArchiveFilterChange?: (showArchievedItems: boolean) => void;
-}> = ({ filters, chartIds, clearCharts, showArchived, onArchiveFilterChange }) => {
+  onlyMyExecutionsFilterState?: {
+    onlyMyExecutionsValue: boolean;
+    isFilterDisabled: boolean;
+    onOnlyMyExecutionsFilterChange: (filterOnlyMyExecutions: boolean) => void;
+  };
```

Comment:
nit: create a separate type/interface for onlyMyState, to use here as:
`onlyMyExecutionsFilterState?: OnlyMyState;`  for better readability